tests/geocoder/test_geocoder.py

```diff
@@ -0,0 +1,403 @@
+"""
+TESTS FOR bbd.geocoder
+"""
+
+from bbd import geocoder as gc
+import pytest
+
+import geopy
+import timeit
+from math import isclose
+
+import pandas as pd
+import json
+from pathlib import Path
+
+# --------------------------------------
+#  ##########   UNIT TESTS   ##########
+# --------------------------------------
+
+#A valid email to use accross tests
+valid_email = "test@bluebonnetdata.org"
+
+# ###### Test get_geocoder ######
+class TestGet_geocoder:
```

Comment:
Yeah I was confused b/c it makes sense for the CamelCase for the class but I wanted a representation of the actual function name in there was sure what to do for that 🤷‍♂️

Reply:
I think in general we want to avoid mixing CamelCase and snake_case (except for the specific use of prefixing/suffixing if that ever seems required). I think it's okay to use TestGetGeocoder and the like for these tests